contextManager.tsx

```diff
@@ -0,0 +1,82 @@
+import { createContext, useState } from 'react';
+import { Card } from './mock_data';
+
+//Interface Type for my 
+export interface user {
```

Comment:
I would change this to upper case to make it clear that it is a type when we are importing it into other parts of the application